model.js

```diff
@@ -112,4 +119,20 @@ const checkCategories = (category) => {
       else return category
    })
 }
-module.exports = {fetchCategories, fetchReviews, fetchReviewById, fetchComments, newComment, updateVotes, fetchAllUsers, checkCategories}
+
+const deleteComment = (commentId) => {
+   if(!+commentId){
+      return Promise.reject({status: 400, msg: 'Bad Request'})
+   }
+   const checkComment = db.query(`SELECT * FROM comments WHERE comment_id = ${commentId}`).then((checkedComment)=>{
+      if(!checkedComment.rowCount){
+         return Promise.reject({status: 404, msg: 'Not Found'})
+      }
+   })
+   const dbQuery = db.query(`DELETE FROM comments WHERE comment_id = ${commentId};`)
+   return Promise.all([dbQuery, checkComment]).then(()=>{
+      return {status: 204, msg: 'No Content'}
```

Comment:
Don't think you need to return anything so you can just return the Promise.all without the then block 👍